src/questions/usdt.yaml

```diff
@@ -0,0 +1,170 @@
+  answerKeys:
+    - D
+  difficultyLevel: Low
+  choices:
+    - content: The allegation of price manipulation
+      key: A
+    - content: The lack of transparency and audits
+      key: B
+    - content: Security and liquidity issues
+      key: C
+    - content: All of the above
+      key: D
+- uuid: 1c7c34d4-166d-4c24-b3c3-be8b4c430d8f
+  type: SingleChoice
+  content: How is USDT meant to be backed according to Tether Limited?
+  hint: NoHint
+  explanation: USDT is meant to be backed 1:1 by the stable asset, typically held in the form of US Treasury bills.
+  answerKeys:
+    - B
+  difficultyLevel: Low
+  choices:
+    - content: 1:1 by a basket of commodities such as gold and oil
+      key: A
+    - content: 1:1 by the stable asset, typically held in the form of US Treasury bills
+      key: B
+    - content: 1:1 by a basket of cryptocurrencies such as Bitcoin and Ethereum
+      key: C
+    - content: 1:1 by a basket of fiat currencies such as the US dollar, Euro, and Japanese yen
+      key: D
+- uuid: 033cab52-dcd8-4672-a80e-d5317a4cf5cc
+  type: SingleChoice
+  content: What was the original name of Tether?
+  hint: NoHint
+  explanation: It was originally named as Realcoin.
+  answerKeys:
+    - C
+  difficultyLevel: Low
+  choices:
+    - content: Bitcoin
+      key: A
+    - content: Litecoin
+      key: B
+    - content: Realcoin
+      key: C
+    - content: USDT
+      key: D
+- uuid: 6be256bf-a6b4-4e3b-bf76-b6942c4075ba
+  type: SingleChoice
+  content: On which network was Tether originally built?
+  hint: NoHint
+  explanation: Tether was originally built on the Bitcoin Network.
+  answerKeys:
+    - D
+  difficultyLevel: Low
+  choices:
+    - content: Ethereum
+      key: A
+    - content: Solana
+      key: B
+    - content: Polygon
+      key: C
+    - content: Bitcoin
+      key: D
+- uuid: c8b21f0e-caf9-47cd-b9c1-056a6517ba02
+  type: SingleChoice
+  content: Why were Tether's Taiwanese banking partners concerned about the stablecoin?
+  hint: NoHint
+  explanation: Tether's Taiwanese banking partners grew concerned about the lack of know-your-customer (KYC) checks and other safeguards as the stablecoin circulated on various blockchains. They ended their relationship with Tether in 2017, which jeopardized Tether's ability to facilitate the conversion of fiat currency to cryptocurrency and vice versa.
+  answerKeys:
+    - D
+  difficultyLevel: Low
+  choices:
+    - content: They were worried about the volatility of the stablecoin
+      key: A
+    - content: They believed that Tether Limited was engaging in market manipulation
+      key: B
+    - content: They were concerned about the security of the stablecoin
+      key: C
+    - content: They were concerned about the lack of know-your-customer (KYC) checks and other safeguards as the stablecoin circulated on various blockchains
+      key: D
```

Comment:
We should add a section about group and adoption and then couple of questions about it